lib/RuntimeLib.lua

```diff
@@ -58,21 +58,22 @@ TS.Symbol_iterator = Symbol("Symbol.iterator")
 local globalModules = script.Parent:FindFirstChild("node_modules")
 
 function TS.getModule(moduleName)
-	local object = getfenv(2).script.Parent
-	if not globalModules then
-		error("Could not find any modules!", 2)
-	end
-	if object:IsDescendantOf(globalModules) then
-		while object.Parent do
+	local object = getfenv(2).script
+	if object:IsDescendantOf(globalModules or error("Could not find any modules!", 2)) then
```

Comment:
I'd rather keep the error in it's own if statement. I think it makes the code more clear.